.nancy-ignore

```diff
@@ -33,3 +33,6 @@ sonatype-2021-1485 until=2022-11-01
 # - sigs.k8s.io/cluster-api@v1.0.5
 # - sigs.k8s.io/controller-runtime@v0.10.3
 CVE-2021-20329 until=2022-11-01
+
+# pkg:golang/github.com/labstack/echo/v4@v4.5.0
+sonatype-2022-5436
```

Comment:
I think you can fix it by upgrading the apimachinery depencency to 0.24.0

Reply:
We'll do the fix in another PR :-)